scripts/s3.py

```diff
@@ -137,12 +146,7 @@ def main():
     assets_metadata = get_assets_metadata(registry_v2.vaults)
 
     for vault in itertools.chain(special, registry_v1.vaults, registry_v2.vaults):
-        try:
-            data.append(wrap_vault(vault, samples, aliases, icon_url, assets_metadata))
-        except ValueError as error:
-            logger.error(error)
-        except PriceError:
-            pass
```

Comment:
This logic is now handled at a more granular level above (inject empty APY instead of completely failing if we can't calculate APY)